python/cudf/cudf/tests/text/test_subword_tokenizer.py

```diff
@@ -16,230 +14,6 @@ def datadir(datadir):
     return os.path.join(datadir, "subword_tokenizer_data")
 
 
-def assert_equal_tokenization_outputs(hf_output, cudf_output):
-    assert (
-        np.sum(hf_output["input_ids"] != cudf_output["input_ids"].get()) == 0
-    )
-    assert (
-        np.sum(
-            hf_output["attention_mask"] != cudf_output["attention_mask"].get()
-        )
-        == 0
-    )
-
-
-@pytest.mark.skip(reason="segfaults")
-@pytest.mark.parametrize("seq_len", [32, 64])
-@pytest.mark.parametrize("stride", [0, 15, 30])
-@pytest.mark.parametrize("add_special_tokens", [True, False])
-@pytest.mark.parametrize("do_lower_case", [True, False])
-def test_subword_tokenize(
-    seq_len, stride, add_special_tokens, do_lower_case, datadir
-):
-    with open(
-        os.path.join(datadir, "test_sentences.txt"), encoding="utf-8"
-    ) as file:
-        input_sentence_ls = [line.strip() for line in file]
-
-    vocab_dir = os.path.join(datadir, "bert_base_cased_sampled")
-
-    transformers = pytest.importorskip("transformers")
```

Comment:
Likewise, if we don't need this dependency to test the new APIs, we can remove this from `dependencies.yaml`

Reply:
Removed